Fixed a bug when broadcasting (#788)
planetarium/libplanet: CHANGES.md

### Bug fixes

>  -  Fixed a bug where `Swarm<T>` broadcast a message to its source peer when

Reviewer: ```suggestion
 -  Fixed a `Swarm<T>`' bug that had broadcasted a message to its source peer when
```

>     guaranteeing minimum peers to broadcast.  [[#788]]

Reviewer: ```suggestion
    the number of peers are not enough (less than the minimum number).  [[#788]]
```

>     the number of peers are not enough (less than the minimum number).  [[#788]]

Reviewer 2: ```suggestion
    the number of peers is not enough (less than the minimum number).  [[#788]]
```
